script.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable import/no-mutable-exports */
+const x = JSON.parse(localStorage.getItem('savedData'));
+let tasks = [];
+
+if (x) {
+  tasks = x;
+}
```

Comment:
- [ ] you can make your code much shoter by using

 `const x = localStorage.getItem('savedData') ? JSON.parse(localStorage.getItem('savedData')) : [];`
 instead of using if statement